nautobot/docs/user-guide/platform-functionality/approval-workflow.md

```diff
@@ -385,3 +394,17 @@ Affected jobs (Names):
 
 - User can create an `ApprovalWorkflowDefinition` using constraints such as `{"job_model__name__in": ["ExampleDryRunJob", "Example Job of Everything", "Export Object List"]}`. See [Create an Approval Workflow Definition with stages](#create-an-approval-workflow-definition-with-stages)
 - However, constraints today are limited to what can be expressed as a single dictionary of field lookups. More complex logic (e.g., `name=A OR (name=B AND status=active)`) is not possible yet.
+
+### Predefined Approval Workflow
+
+As alluded to, in 3.0, there will be seed data for ths following use cases.
+
+- Permissions created for the 3 identified personas.
+- Groups create for the 3 identified personas, with permissions applied.
+- An Approval Workflow created that leverages the approval persona defined.
+
+In order to get like for like workflow, you should do the following:
+
+1. Update the Approval Workflow `Model Constraints` to represent the jobs you wish to control.
+2. Add users that initially run the job as `nautobot-default-scheduledjobs-operator`.
+3. Add users that approve the job as `nautobot-default-scheduledjobs-approver`.
```

Comment:
We should probably call this out in the "Administrator actions" section of the 3.0 release notes as well.